app/assets/stylesheets/custom.css.scss

```diff
@@ -108,3 +108,46 @@ footer {
 
 }
 
+/* sidebar */
+
+aside {
+
+  section.user_info {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `user_info` should be written in lowercase with hyphens